### ISSUE TITLE
Feature Idea: Moving cards to other boards

### ISSUE THREAD

**Issue by obvionaoe** [mattermost/focalboard#3253]
## Summary

Allow cards to be moved to another board

## How important this is to me and why

Importance: High

We constantly need to move cards to other boards

Use cases:
1. When a card was created in a board, but then that project grew and warranted the creation of a new board, so you'll need to move that card to the new board.
2. You just created a card in the wrong board

## Additional context/similar features

Move the card to the board and add all of the cards properties to the board it was moved to.

**Comment by kibblewhite** [comment]
Yes, totally this. I've also been looking in the documentation to see if it was something I missed, but I could not find anything... yet.

I believe someone else has also asked for something similar here:
https://github.com/mattermost/focalboard/discussions/886

**Comment by obvionaoe** [comment]
Any updates?

**Comment by skaldesh** [comment]
Without this feature, boards are just not usable for us

**Comment by Saya47** [comment]
Please consider this!

**Comment by artromone** [comment]
REALLY NEED THIS!!!
